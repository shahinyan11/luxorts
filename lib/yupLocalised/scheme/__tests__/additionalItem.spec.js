import schema from '../additionalItem';

describe('Additional item validation schema', () => {
  it('matches snapshot', () => {
    expect(schema.describe()).toMatchSnapshot();
  });
});
