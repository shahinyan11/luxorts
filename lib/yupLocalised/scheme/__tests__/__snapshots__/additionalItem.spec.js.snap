// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Additional item validation schema matches snapshot 1`] = `
Object {
  "fields": Object {
    "additional": Object {
      "fields": Object {
        "description": Object {
          "label": undefined,
          "meta": undefined,
          "notOneOf": Array [],
          "oneOf": Array [],
          "tests": Array [],
          "type": "string",
        },
        "name": Object {
          "label": undefined,
          "meta": undefined,
          "notOneOf": Array [],
          "oneOf": Array [],
          "tests": Array [],
          "type": "string",
        },
      },
      "label": undefined,
      "meta": undefined,
      "notOneOf": Array [],
      "oneOf": Array [],
      "tests": Array [],
      "type": "object",
    },
  },
  "label": undefined,
  "meta": undefined,
  "notOneOf": Array [],
  "oneOf": Array [],
  "tests": Array [],
  "type": "object",
}
`;
